src/ray/object_manager/plasma/eviction_policy.h

```diff
@@ -172,13 +172,12 @@ class EvictionPolicy {
   /// The number of bytes pinned by applications.
   int64_t pinned_memory_bytes_;
 
-  /// Pointer to the plasma store info.
-  PlasmaStoreInfo *store_info_;
-
-  const IAllocator &allocator_;
-
   /// Datastructure for the LRU cache.
   LRUCache cache_;
+
+  const ObjectStore &object_store_;
```

Comment:
Consider using smart pointers instead of raw references to avoid destruction order bugs.